sdk/cargo-build-bpf/tests/crates/noop/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+//! @brief Example Rust-based BPF noop program
+
+extern crate solana_program;
+use solana_program::{
+    account_info::AccountInfo, entrypoint, entrypoint::ProgramResult, pubkey::Pubkey,
+};
+
+entrypoint!(process_instruction);
+#[allow(clippy::unnecessary_wraps)]
```

Comment:
What's this lint for?

Reply:
left over from programs/rust/noop. I'll remove it.